homeassistant/components/media_extractor/strings.json

```diff
@@ -13,6 +13,20 @@
           "description": "The type of the content to play. Must be one of MUSIC, TVSHOW, VIDEO, EPISODE, CHANNEL or PLAYLIST MUSIC."
         }
       }
+    },
+    "extract_media": {
```

Comment:
I don't think `extract_media` is a good name for the service. The gist of the service is that it returns a dictionary with a URL with the wanted format, can we try to make that clear in the name too? Maybe "get_media_url" or "get_stream_url" or something like that?